### PR TITLE
fix(crux): move platform detection after priviledge escalation in install.docker.hbr

### PR DESCRIPTION
**Pull request by m8vago**

### CODE REVIEW

File: web/crux/install-docker.sh.hbr [hunk 1]
```diff
@@ -87,7 +84,7 @@ set_environment() {
     if [ -z ${DOCKER_HOST:-} ]; then
       HOST_DOCKER_SOCK_PATH="/var/run/docker.sock"
     else
-      if [ ${DOCKER_HOST:0:7} = "unix://" ]; then
+      if [ ${DOCKER_HOST:0:7} == "unix://" ]; then
```
Comment (polaroi8d) [inline web/crux/install-docker.sh.hbr:87]: 🦅

**Review by polaroi8d — APPROVED** [review]
LGTM works on MacOS

**Review by c3ppc3pp — APPROVED** [review]

**Review by balanceee — CHANGES_REQUESTED** [review]
Not working on windows:
Installation process needs root privileges

**Comment by m8vago** [comment]
> Not working on windows: Installation process needs root privileges

Thanks! Fixed and tested on @robot9706 's machine

**Review by balanceee — APPROVED** [review]

**Pull request merged**